x-pack/plugins/secops/public/components/draggables/index.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiBadge, IconType } from '@elastic/eui';
+import * as React from 'react';
+import { pure } from 'recompose';
+import styled from 'styled-components';
+
+import { escapeQueryValue } from '../../lib/keury';
+import { DragEffects, DraggableWrapper } from '../drag_and_drop/draggable_wrapper';
+import { escapeDataProviderId } from '../drag_and_drop/helpers';
+import { Provider } from '../timeline/data_providers/provider';
+
+export interface DefaultDraggableType {
+  id: string;
+  field: string;
+  value?: string | null;
```

Comment:
An interesting side effect of documenting the fact that `value` may be undefined via `value?` as opposed to `value: string | null | undefined` is that it's possible to use `DefaultDraggable` while omitting the `value` prop, like this test: https://github.com/elastic/kibana/pull/33370/files#diff-c73b70d456195cc298b62951694789a2R60
, but that doesn't seem like a valid use case. If you agree, perhaps, `value: string | null | undefined`, or a variation on that theme might work?

Reply:
Yea, I should use the explicit `undefined` as it is not suppose to be optional but rather if the code has an undefined value it just doesn't render anything.